pandas/core/arrays/datetimelike.py

```diff
@@ -566,9 +566,11 @@ def view(self, dtype=None):
 
         Returns
         -------
-        ndarray
```

Comment:
doc-string

Reply:
can you elaborate on what you're asking for here?

Reply:
see my comments above, just remove the doc-string entirely